src/Models/Node.ts

```diff
@@ -32,6 +33,10 @@ export class Node extends Model {
             description: {
                 type: DataTypes.STRING,
                 allowNull: true
+            },
+            notifications: {
```

Comment:
maybe we should use better naming that sounds like boolean like `hasEnabledNotifications` or just `enabledNotifications`